modules/cloudtrail/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "company_id" {
+  type        = string
+  description = "Company ID obtained from KSOC"
+}
+
+variable "lambda_debug" {
+  type        = bool
+  default     = false
+  description = "More verbose logging in ksoc-observer Lambda"
+}
+
+variable "lambda_timeout" {
+  type        = string
+  default     = "30s"
+  description = "HTTP timeout for ksoc-observer Lambda"
+}
+
+variable "ingest_url" {
+  type        = string
+  description = "URL of KSOC API"
+}
+
+variable "resources_prefix" {
+  type        = string
+  default     = "ksoc-discovery"
+  description = "All resources created by this module will begin with this name"
+}
```

Comment:
I think this should be removed we should be hard-coding this where required, people shouldn't be able to easily override it. It will lead to inconsistencies when performing root cause analysis.

Reply:
See @benhartley comment about it :) If someone wants to try this module twice on the same AWS account, it wouldn't be possible.